examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -25,25 +21,18 @@ import (
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/actions"
 	"github.com/ava-labs/hypersdk/examples/morpheusvm/consts"
 	"github.com/ava-labs/hypersdk/rpc"
-	"github.com/ava-labs/hypersdk/utils"
+	"github.com/ava-labs/hypersdk/tests/e2e"
+	"github.com/ava-labs/hypersdk/tests/workload"
 
-	runner_sdk "github.com/ava-labs/avalanche-network-runner/client"
 	lrpc "github.com/ava-labs/hypersdk/examples/morpheusvm/rpc"
 	ginkgo "github.com/onsi/ginkgo/v2"
 )
 
-const (
-	startAmount = uint64(10000000000000000000)
-	sendAmount  = uint64(5000)
-
-	healthPollInterval = 3 * time.Second
-)
-
-func TestE2e(t *testing.T) {
-	ginkgo.RunSpecs(t, "morpheusvm e2e test suites")
-}
-
 var (
+	// TODO
```

Comment:
TODO: for this PR - skip ginkgo update run script to focus on only the `HyperSDK APIs` tests, and leave the network running if `MODE=run`.

As a follow up to this PR, we should convert to using `tmpnet` from AvalancheGo and use `--reuse-network` flag + a much simplified run script to leave the network running.

Reply:
If `MODE=run` the run script will now add `ginkgo.focus=Ping` to skip the rest of the test suite